Task2/Locators.cs

```diff
@@ -0,0 +1,91 @@
+﻿namespace Task2
+{
+    class Locators
+    {
+        /// <summary>
+        /// Locators for enter in Mail 
+        /// </summary>
+        public class LoginMailLocators
+        {
+            public string PageLocator { get; } = "https://mail.ru/";
+            public string EnterLoginLocator { get; } = "//input[@name = 'login']";
+            public string EnterPasswordLocator { get; } = "//input[@name = 'password']";
+            public string EnterButtonLocator { get; } = "//*[text() = 'Войти']";
+        }
+
+        /// <summary>
+        /// Locators for writing message and send it
+        /// </summary>
+        public class SendingMailMessageLocators
+        {
+            public string WritingMailButtonLocator { get; } = "//*[text() = 'Написать письмо']";
```

Comment:
to become attached to the text is only in the most hopeless situations.

If my browser is on UK locale this tests will fail